parsl/executors/base.py

```diff
@@ -87,6 +90,98 @@ def scaling_enabled(self) -> bool:
         """
         pass
 
+    @abstractmethod
+    def status(self) -> Dict[Any, JobStatus]:
+        """Return the status of all jobs/blocks currently known to this executor.
+
+        :return: a dictionary mapping job ids to status strings
+        """
+        pass
+
+    @property
+    @abstractmethod
+    def status_polling_interval(self) -> int:
+        """Returns the interval, in seconds, at which the status method should be called. The
+        assumption here is that, once initialized, an executor's polling interval is fixed.
+        In practice, at least given the current situation, the executor uses a single task provider
+        and this method is a delegate to the corresponding method in the provider.
+
+        :return: the number of seconds to wait between calls to status() or zero if no polling
+        should be done
```

Comment:
is there a situation where no polling should be done but error_management_enabled is True?

Reply:
polling is also used by strategies, so the two are separate, at least logically.